api/allIncidents/incidentsModel.js

```diff
@@ -18,7 +18,9 @@ module.exports = {
  * Returns all incidents in the db sorted by newest incident first
  */
 async function getIncidents() {
-  return await db('incidents').whereNot({ date: null }).orderBy('date', 'desc');
+  return await db('incidents')
+    .whereNot({ date_created: null })
+    .orderBy('date', 'desc');
```

Comment:
This will need to be `description` rather than `desc` once the DS table is put in place, yes?

Reply:
This desc means descending

Reply:
Ah, my mistake, worse than a Ctrl+F Thanks!